<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HorizonManager</class>
 <widget class="QWidget" name="HorizonManager">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>609</width>
    <height>449</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <property name="sizeConstraint">
      <enum>QLayout::SetDefaultConstraint</enum>
     </property>
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>Regions</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QPushButton" name="addRegionB">
            <property name="minimumSize">
             <size>
              <width>32</width>
              <height>32</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>32</width>
              <height>32</height>
             </size>
            </property>
            <property name="toolTip">
             <string>Add Region</string>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="removeRegionB">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="minimumSize">
             <size>
              <width>32</width>
              <height>32</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>32</width>
              <height>32</height>
             </size>
            </property>
            <property name="toolTip">
             <string>Remove Region</string>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="saveB">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="minimumSize">
             <size>
              <width>32</width>
              <height>32</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>32</width>
              <height>32</height>
             </size>
            </property>
            <property name="toolTip">
             <string>Save Regions</string>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QListView" name="regionsList"/>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="title">
        <string>Points</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QPushButton" name="addPointB">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="minimumSize">
             <size>
              <width>32</width>
              <height>32</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>32</width>
              <height>32</height>
             </size>
            </property>
            <property name="toolTip">
             <string>Add Point</string>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="removePointB">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="minimumSize">
             <size>
              <width>32</width>
              <height>32</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>32</width>
              <height>32</height>
             </size>
            </property>
            <property name="toolTip">
             <string>Remove Point</string>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="clearPointsB">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="minimumSize">
             <size>
              <width>32</width>
              <height>32</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>32</width>
              <height>32</height>
             </size>
            </property>
            <property name="toolTip">
             <string>Clear all points</string>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QLabel" name="polygonValidatoin">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="minimumSize">
             <size>
              <width>32</width>
              <height>32</height>
             </size>
            </property>
            <property name="toolTip">
             <string/>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="selectPointsB">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="minimumSize">
             <size>
              <width>32</width>
              <height>32</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>32</width>
              <height>32</height>
             </size>
            </property>
            <property name="toolTip">
             <string>Select points from the sky map</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="checkable">
             <bool>true</bool>
            </property>
            <property name="flat">
             <bool>false</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QTableView" name="pointsList"/>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="1,5">
     <item>
      <widget class="QLabel" name="tipLabel">
       <property name="minimumSize">
        <size>
         <width>64</width>
         <height>64</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>64</width>
         <height>64</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Artificial horizon is used to define &lt;span style=&quot; font-weight:600;&quot;&gt;Regions&lt;/span&gt; that are &lt;span style=&quot; font-style:italic;&quot;&gt;blocked&lt;/span&gt; from view from your vantage point (e.g. Tall trees or buildings). To draw a &lt;span style=&quot; font-weight:600;&quot;&gt;Region&lt;/span&gt;, define a list of &lt;span style=&quot; font-weight:600;&quot;&gt;Points&lt;/span&gt; (&lt;span style=&quot; font-style:italic;&quot;&gt;Minimum&lt;/span&gt; 5) that encompasses the blocked area. Add the points manually, or by selecting them from the Sky Map after clicking &lt;span style=&quot; font-style:italic;&quot;&gt;Select Points&lt;/span&gt; button. &lt;span style=&quot; font-weight:600;&quot;&gt;First&lt;/span&gt; and &lt;span style=&quot; font-weight:600;&quot;&gt;Last&lt;/span&gt; points must be on the horizon. Polygons must be closed to be considered valid regions.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
